src/Libraries/Microsoft.Extensions.AI.Abstractions/AudioTranscription/AudioTranscriptionOptions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>Represents the options for an audio transcription request.</summary>
+public class AudioTranscriptionOptions
+{
+    private CultureInfo? _audioLanguage;
+
+    /// <summary>Gets or sets the completion ID for the audio transcription.</summary>
+    /// <remarks>Long running jobs may use this ID for status pooling.</remarks>
+    public string? CompletionId { get; set; }
+
+    /// <summary>Gets or sets the model ID for the audio transcription.</summary>
+    public string? ModelId { get; set; }
+
+    /// <summary>Gets or sets the language for the audio transcription.</summary>
+    public string? AudioLanguage
```

Comment:
Is this the language that's in the audio or the desired language for the resulting transcription, i.e. it might be transcribed and translated?

Reply:
This is the language of the Audio (input). 
Initially I was using `Source` in the name, but everything that has `Audio` in the name made sense for me that where refering the the audio format (in this context only the input)

Reply:
Will update the PR to use `ISpeechToText` so I can use `SpeechLanguage`, will make it more clear.